### DISCUSSION TITLE
Setting limits for seed ratio/time on cross-seeds

### DISCUSSION THREAD

**Discussion by xiNeRoar** [cross-seed/cross-seed#509] [Q&A] [answered]
may i ask when a torrent is be cross-seeding, how long will it last?
is there anyway i can control the seed ratio or seed time of a cross-seeding torrent?

**Comment by zakkarry** [comment]
Your torrent client should allow for settings for this, this has nothing to do with cross-seed in its current implementation. Set either a label's seed ratio/time settings or set global ones that will apply to all torrents.

**Comment by spyhunter493** [comment, marked as answer]
[https://github.com/StuffAnThings/qbit_manage](url)

**Comment by xiNeRoar** [comment]
Thanks
